src/occupational_classification/utils/soc_data_access.py

```diff
@@ -6,10 +6,33 @@
 import pandas as pd
 
 
+def combine_job_title(row: pd.DataFrame) -> str:
+    """Produces full job title wih IND and ADD qualifiers.
+
+    Args:
+        row: A row containing job title, IND and ADD qualifiers
```

Comment:
Can you please add type hints to Args (str, int, tuple, etc). For example:
row(str): ...